django-stubs/test/runner.pyi

```diff
@@ -55,6 +55,7 @@ class RemoteTestResult:
     def stopTestRun(self) -> None: ...
     def startTest(self, test: Any) -> None: ...
     def stopTest(self, test: Any) -> None: ...
+    def addDuration(self, test: Any, elapsed: Any) -> None: ...
```

Comment:
This could be better typed, but will be easier to handle fixing up `RemoteTestResult` as a whole.